Automattic/wp-calypso
# Jetpack Licensing: Add the foundation for the Jetpack Licensing Portal

Context:
- pbtFPC-118-p2
- pbtFPC-103-p2

The purpose of this PR is to prepare for the work that is soon to be done as part of the Licensing Portal project.
At this stage, we are looking for general code style and organization feedback so we can have a solid foundation to build the Licensing Portal on.

#### Changes proposed in this Pull Request

* Add the Jetpack Licensing Portal foundation to the **development environment only**.

#### Testing instructions
* Make sure the latest code style, organization and modularization practices are employed.
* The portal is a new section available at `http://jetpack.cloud.localhost:3000/partner-portal`.
* The current functionality of selecting a key is mostly for demo/development purposes only - it will not be preserved in exactly the state that it is right now.
* Please request a partner key from team Infinity in order to be able to use the portal.
